test/com/google/cose/structure/SignStructureTest.java

```diff
@@ -0,0 +1,30 @@
+package com.google.cose.structure;
+
+import co.nstant.in.cbor.model.ByteString;
+import co.nstant.in.cbor.model.Map;
+import co.nstant.in.cbor.model.NegativeInteger;
+import co.nstant.in.cbor.model.UnsignedInteger;
+import com.google.cose.TestUtilities;
+import com.google.cose.structures.SignStructure;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class SignStructureTest {
+  @Test
+  public void testHappyCase() {
```

Comment:
Shouldn't we also test sad cases? (this is a general comment. feel free to add more tests later for negative paths)